### PR TITLE
Improve debugging interfaces documentation for .NET Framework

### PR DESCRIPTION
**Pull request by hoyosjs**
## Summary

Adds further documentation for .NET Framework's debugging API. 

cc: @tommcdon @noahfalk @cshung

### CODE REVIEW

File: docs/framework/unmanaged-api/debugging/dacpmethoddescdata-structure.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| `moduleInfo`                 | Pointer to a buffer used internally by the runtime for module information.                      |
+| `MDToken`                    | Token associated with the given method.                                                         |
+| `payloadGC`                  | Pointer to a garbage collection buffer used internally by the runtime.                          |
+| `payloadGC2`                 | Pointer to a garbage collection buffer used internally by the runtime.                          |
+| `managedDynamicMethodObject` | If the method is dynamic, the runtime uses this buffer internally for information tracking.     |
+| `requestedIP`                | Used to populate the structure per request when given a native code address.                    |
+| `rejitDataCurrent`           | Information about the latest instrumented version the method.                                   |
+| `rejitDataRequested`         | Rejit information for the requested native address.                                             |
+| `cJittedRejitVersions`       | Number of                                                                                       |
```
Comment (tommcdon) [inline docs/framework/unmanaged-api/debugging/dacpmethoddescdata-structure.md]: Incomplete sentence

**Review by rpetrusha — CHANGES_REQUESTED** [review]
This looks really good, @hoyosjs. I've left some comments and suggestions.

**Review by hoyosjs — COMMENTED** [review]
@rpetrusha Thanks for the review. I think I addressed most of your comments, unless what was in said documents was as intended.

**Review by tommcdon — APPROVED** [review]

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks both for creating this documentation and addressing my comments, @hoyosjs. I'll merge your PR now. It should be live on docs.microsoft.com in the next day or two.

**Pull request merged**